src/get-started/java-binding.md

```diff
@@ -604,13 +613,14 @@ The TestKit also allows creating blocks that contain all current [in-pool][in-po
 transactions:
```

Comment:
The clause with the example was suggested to be added to this sentence to clarify when in-pool transactions might be added.